src/supermarket/db/migrate/20160914215547_move_quality_metric_results.rb

```diff
@@ -0,0 +1,26 @@
+class MoveQualityMetricResults < ActiveRecord::Migration
+  def change
+    foodcritic_qm = QualityMetric.create!(name: 'Foodcritic')
+    collab_num_qm = QualityMetric.create!(name: 'Collaborator Number')
+
+    CookbookVersion.all.each do |cookbook_version|
```

Comment:
At the risk of triggering the need to test the migration of production data again, consider:

```diff
- CookbookVersion.all.each do |cookbook_version|
+ CookbookVersion.where("foodcritic_failure IS NOT NULL or collaborator_failure IS NOT NULL").each do |cookbook_version|
```

The difference in production is substantial:

```
> CookbookVersion.count
=> 16855
> CookbookVersion.where("foodcritic_failure IS NOT NULL or collaborator_failure IS NOT NULL").count
=> 9447
```